app/src/main/java/org/mozilla/fenix/components/FxaServer.kt

```diff
@@ -6,30 +6,23 @@ package org.mozilla.fenix.components
 import android.content.Context
 import mozilla.components.service.fxa.ServerConfig
 import mozilla.components.service.fxa.ServerConfig.Server
-import org.mozilla.fenix.FeatureFlags
 import org.mozilla.fenix.ext.settings
 
 /**
  * Utility to configure Firefox Account servers.
  */
 
 object FxaServer {
-    const val CLIENT_ID = "a2270f727f45f648"
-    const val REDIRECT_URL = "https://accounts.firefox.com/oauth/success/$CLIENT_ID"
+    private const val CLIENT_ID = "a2270f727f45f648"
 
-    @Suppress("ConstantConditionIf", "UNUSED_PARAMETER")
-    fun redirectUrl(context: Context) = if (FeatureFlags.asFeatureWebChannelsDisabled) {
-        REDIRECT_URL
-    } else {
-        "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
-    }
+    fun redirectUrl() = "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
```

Comment:
This should be a constant
```suggestion
    const val REDIRECT_URL = "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
```

Reply:
Wait :) it's used in somewhere as function.

Reply:
https://github.com/mozilla-mobile/fenix/blob/36b1c710e7cf2612a8d28585f301ec1140381710/app/src/main/java/org/mozilla/fenix/components/Services.kt#L29
Should we hardcode it or go with function?

Reply:
Done.